.github/workflows/release.yaml

```
@@ -0,0 +1,80 @@
+---
+name: Release
+
+run-name: release ${{ inputs.tag }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to be released'
+        required: true
+
+permissions:
+  packages: write
+  id-token: write
+  contents: read
+
+jobs:
+  publish:
+    env:
+      VERSION: ${{ inputs.tag }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set env vars
+        run: |
+          echo PKG_VERSION="$(jq -r .version < package.json)" >> $GITHUB_ENV
+          echo PKG_NAME="$(jq -r .name < package.json)" >> $GITHUB_ENV
+          echo PKG_BASENAME="$(basename $(jq -r .name < package.json))" >> $GITHUB_ENV
+      - name: Ensure the branch is protected
+        run: |
+          if [[ "${{ github.ref_protected }}" == "false" ]]; then
```

```suggestion
          if [[ "${{ github.ref_protected }}" = "false" ]]; then
```

This one as well to be consistent with the check below on version